CMGTools/TTHAnalysis/python/plotter/mcAnalysis.py

```diff
@@ -195,6 +205,8 @@ def getYields(self,cuts,process=None,nodata=False,makeSummary=False,noEntryLine=
             from multiprocessing import Pool
             pool = Pool(self._options.jobs)
             retlist  = pool.map(_runYields, tasks)
+            pool.close()
+            pool.join()
```

Comment:
Why do we need this? from the docs, pool.map already blocks until the tasks are done successfully ( https://docs.python.org/2/library/multiprocessing.html#module-multiprocessing.pool )
is it to solve the hangs that sometimes happen?